test/controllers/AccountSummaryControllerSpec.scala

```diff
@@ -85,7 +87,8 @@ lazy implicit val messages: MessagesImpl = MessagesImpl(Lang("en-GB"), messagesA
       mockDetailsService,
       mockDataCacheConnector,
       mockDateService,
-      mockServiceInfoService
+      mockServiceInfoService,
+      injview
```

Comment:
Can we rename this to something better. Like "injectedViewInstance". Something which helps the code be more self documenting.